driver/pace/driver/driver.py

```diff
@@ -675,7 +674,7 @@ def cleanup(self):
         )
         if (
             self.comm.Get_size()
-            >= self.config.performance_config.json_all_rank_threshold
+            <= self.config.performance_config.json_all_rank_threshold
```

Comment:
Well the idea was to not save the full performance .json when you are running a high numbers to evade writing a several GB json file. We save a rank 0 json, which arguably is enough since the model is so tightly coupled